### ISSUE TITLE
PUT is non-available in buildRequest() object

### ISSUE THREAD

**Issue by traynaud** [microsoftgraph/msgraph-sdk-java#319]
According to the documentation of the Team creation in  graph ( https://docs.microsoft.com/fr-fr/graph/api/team-put-teams?view=graph-rest-1.0&tabs=java), the type of expected HTTP request is a PUT (PUT /groups/{id}/team)

Thus, an example is given at the bottom of the page:
graphClient.groups("{id}").team().buildRequest().put(team);

However, the PUT method does not exist in the interface ITeamRequest.

**Comment by ciadti** [comment]
how to solve this problem   graphClient.groups("{id}").team() 	.buildRequest() 	.put(team);

PUT is non-available

**Comment by Romadelf** [comment]
@ciadti As of now and as far as I'm concerned, there is no build-in sdk method performing this at the moment.

However, thanks to this https://github.com/microsoftgraph/msgraph-sdk-java/pull/206 merged pull request (available since 1.1.0.), I assume the best workaround we currently have is doing the following:
```
String url = String.format("/groups/%s/team", groupID);
Team createdTeam = graphClient.customRequest(url, Team.class).buildRequest().put(team);
```

**Comment by baywet** [comment]
thank you for your patience on the matter. I just authored PR #446 which fixes the issue.
This issue will be closed when the PR gets merged.
